### PR TITLE
Ad5933 feature update

### PR DESCRIPTION
**Pull request by pmallick-adi**
V2: 

- Added ad5933_get_data function
- Fixed style inconsistencies
- Added null pointer checks
- Rolled back radToDeg180ovPi and M_PI definitions

### CODE REVIEW

File: drivers/impedance-analyzer/ad5933/ad5933.c [hunk 1]
```diff
@@ -457,3 +457,31 @@ double ad5933_calculate_impedance(struct ad5933_dev *dev,
...
+{
+	if ((multiplier != AD5933_SETTLING_X2) && (multiplier != AD5933_SETTLING_X4))
+		multiplier = AD5933_SETTLING_X1;
+
+	ad5933_set_register_value(dev,
+				  AD5933_REG_SETTLING_CYCLES,
+				  number_cycles | (multiplier << 9),
+				  2);
+	/* Store the last settings made. */
+	dev->current_settling = number_cycles;
```
Comment (mchindri) [inline drivers/impedance-analyzer/ad5933/ad5933.c:492]: As I see in your changes, current_settiling is not used anywhere. Can you remove it?

**Review by mchindri — APPROVED** [review]
It looks good, just a small comment

**Review by amiclaus — APPROVED** [review]

File: drivers/impedance-analyzer/ad5933/ad5933.c (hunk 1 above)
Comment (pmallick-adi) [reply drivers/impedance-analyzer/ad5933/ad5933.c:492]: Shall I remove this as part of new commit or force push the update ?

Comment (mchindri) [reply drivers/impedance-analyzer/ad5933/ad5933.c:492]: Forcee push,  and also from ad5933_dev and ad5933_init_param

Comment (pmallick-adi) [reply drivers/impedance-analyzer/ad5933/ad5933.c]: Done.

**Review by dbogdan — APPROVED** [review]

**Pull request merged**